src/components/NavigationSection/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -25,8 +25,9 @@ exports[`NavigationSection component renders correctly 1`] = `
           d="m7 14 5-5 5 5z"
         />
       </svg>
-    </div>
+    </button>
     <div
+      role="region"
```

Comment:
I wasn't familiar with this role. What exactly it entitles to?

Reply:
It’s used to demarcate an aria of significance but where a more semantic tag doesn’t really work.